build.config.js

```diff
@@ -0,0 +1,12 @@
+const serverConfig = {
+  entryPoints: ['src/minesweeper/infrastructure/server/server'],
+  bundle: true,
+  platform: 'node',
+  target: 'node14', //depend of the deploy server
```

Comment:
If we change node version should we change this?

Reply:
yes